tests/models/test_vision.py

```diff
@@ -6,33 +6,32 @@
 from pl_bolts.datasets import DummyDataset
 from pl_bolts.models import GPT2, ImageGPT, SemSegment, UNet
 
+pl.seed_everything(42)
 
-def test_igpt(datadir):
-    pl.seed_everything(0)
```

Comment:
As long as we keep this seed, `test_igpt` passes with the current loss value `1.7`, right?

Reply:
Yes, but that's unreliable testing, I mean the loss should rather satisfy some value irrespective of seed.

Reply:
Yeah, you're right... but I'm still not sure if specifying a certain value of the loss is reasonable because the value is based on the implementation which we want to test, right...? But for now, I guess we could just change this value.